packages/transport/src/modules/accounts/data/IdentityDeletionProcess.ts

```diff
@@ -0,0 +1,101 @@
+import { serialize, type, validate } from "@js-soft/ts-serval";
+import { nameof } from "ts-simple-nameof";
+import { CoreDate, CoreId, CoreSynchronizable, ICoreSynchronizable } from "../../../core";
+import { IdentityDeletionProcessStatus } from "./IdentityDeletionProcessStatus";
+
+export interface IdentityDeletionProcessJSON extends Omit<ICoreSynchronizable, "id"> {
+    id: string;
+    status: IdentityDeletionProcessStatus;
+    createdAt?: string;
+    createdByDevice?: string;
+    approvedAt?: string;
+    approvedByDevice?: string;
+    gracePeriodEndsAt?: string;
+    cancelledAt?: string;
+    cancelledByDevice?: string;
+    rejectedAt?: string;
```

Comment:
As with the order of the fields of `IdentityDeletionProcessStatus` (the field `Rejected` is placed before the field `Approved`), I would also place the two fields `rejectedAt?` and `rejectedByDevice?` before `approvedAt?` here, to avoid confusion with the `cancelledAt?` and `cancelledByDevice?` fields. Same below and in the IdentityDeletionProcessMapper and the IdentityDeletionProcessDTO.